argocd/overlays/moc-infra/applications/envs/octo/gohan/cluster-resources.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: cluster-resources
+  annotations:
+    notifications.argoproj.io/subscribe.on-sync-succeeded.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-running.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-failed.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: "argocd-notifications"
+    notifications.argoproj.io/subscribe.on-sync-degraded.slack: "argocd-notifications"
```

Comment:
can you remove these, this enables argocd notifications for this app in slack and I don't think we're ready to enable these for other apps.

Reply:
sounds good 
removed :+1: